core/templates/domain/bottom_navbar/bottom-navbar-status.service.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service for maintaining the open/closed status of the
+ * bottom navigation bar.
+ */
+
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+
+import { WindowDimensionsService } from
+  'services/contextual/window-dimensions.service';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class BottomNavbarStatusService {
+  constructor(private windowDimensionsService: WindowDimensionsService) {}
+    bottomNavbarIsEnabled: boolean = false;
```

Comment:
Should we define this before the constructor?

Reply:
DOne